staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/generated.proto

```diff
@@ -25,10 +25,42 @@ import "k8s.io/api/authentication/v1/generated.proto";
 import "k8s.io/apimachinery/pkg/apis/meta/v1/generated.proto";
 import "k8s.io/apimachinery/pkg/runtime/generated.proto";
 import "k8s.io/apimachinery/pkg/runtime/schema/generated.proto";
+import "k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/generated.proto";
 
 // Package-wide variables from generator "generated".
 option go_package = "v1alpha1";
 
+// AuditConfiguration represents a cluster level audit configuration
+message AuditConfiguration {
+  // +optional
+  optional k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
+
+  // Policy to enforce on the given backend
+  // if undefined it will default to the statically configured cluster policy if available
```

Comment:
> if undefined it will default to the statically configured cluster policy if available

What will happen if the policy is defined but has zero rules set?
What will happen if the policy is defined and OmitStages is set but has zero rules set?